### ISSUE TITLE
BSOD on Windows 7 x64 with ManualMap Injection

### ISSUE THREAD

**Issue by Mecanik** [DarthTon/Blackbone#332]
Just an FYI, maybe you are not aware... but I tested on numerous Win 7 x64 machines to inject x64 process using Kernel -> ManualMap and every time I got BSOD.

The driver is compiled in release mode, using the latest SDK/WDK etc.

Could I provide you something that would help in resolving this issue ?

Many thanks

**Comment by DarthTon** [comment]
BSOD memory dump and blackbone driver binary + pdb symbols will help.

**Comment by Mecanik** [comment]
Yes will do, right now I'm finishing something urgent. PS: In Debug mode the driver does not crash... o.O
